src/webhooks/webhooks.js

```diff
@@ -0,0 +1,13 @@
+/* globals Buffer */
+const Base = require('../base');
+const VerifyHeader = require('./resources/verify_header');
+
+class Webhooks extends Base {
+  constructor(...params) {
+    super(...params);
+    const verifyHeader = new VerifyHeader(this);
+    this.verifyHeader = verifyHeader.verify.bind(verifyHeader);
```

Comment:
This deviates from the current pattern with `Video` and `Data`.

To stay consistent, this would change to:

```
this.VerifyHeader = new VerifyHeader(this);
```

And the public API would be:

```
const { Webhooks } = new Mux(accessToken, secret);
Webhooks.VerifyHeader.verify(body, signature, secret);
```

That seemed a little verbose, so I decided to make this shortcut. LMK if you have a better idea.